### DISCUSSION TITLE
how to cross compiling to target x86_64-unkown-linux-musl?

### DISCUSSION THREAD

**Discussion by reison1218** [sagebind/isahc#345] [Questions] [answered]
when i cross compiling to the x*6_64-unknown-linux-musl,it fail.
the error message: failed to run custom build command for `libz-sys v1.1.3`

could it support x86_64-unkown-linux-musl？
if support.how to compiling to it?
pls help,thanks

**Comment by sagebind** [comment, marked as answer]
Can you share the full error message? It sounds like you're probably missing some packages on your system, maybe such as musl gcc and libz development headers. It's hard to say without the full error message.

Note also that it is difficult to cross-compile to musl when using OpenSSL because OpenSSL often dynamically links to musl instead of statically linking, which is a bit odd. You can enable the `static-ssl` feature if you have this problem.

**Reply by reison1218** [reply]
thanks for help,when i enable the static-ssl,it's work!
